### ISSUE TITLE
Show when was the last date we updated the meeting page from the Knesset API

### ISSUE THREAD

**Issue by morchickit** [hasadna/Open-Knesset#653]
Another piece of information that I think is crucial for users to know and can explain how updated is the data

**Comment by alonisser** [comment]
@morchickit Perhaps some kind of icon and not an exact date (or both)?

**Comment by morchickit** [comment]
I think both would be helpful.

**Comment by nivm** [comment]
@morchickit 
Is that what you meant ?

![screenshot from 2016-06-13 20 37 50](https://cloud.githubusercontent.com/assets/1131350/16017058/d819d8e8-31a6-11e6-98d7-a6216b5a318f.png)

**Comment by alonisser** [comment]
@nivm thanks!  where is the date actually from? is it when we inserted or protocol date?

**Comment by nivm** [comment]
@alonisser 
I have added a field for update time of the meeting.

The `CommitteeMeeting` model has 2 update dates. (`protocol_text_update_date`, `protocol_parts_update_date`).
but the `CommitteeMeeting` can be updated by different operations like `add-lobbyist`, `mk`.

**Comment by alonisser** [comment]
@nivm That's what I suspected. But I'm not sure this is correct. @morchickit should say, what is the required "update date" here? a protocol update? (the latest from the two existing fields) or any change or update in the data, like adding an annotation, lobbyist, etc?

**Comment by nivm** [comment]
@morchickit - Can you please elaborate?

**Comment by morchickit** [comment]
No, not any type of the data, just when was the last protocol added to Oknesset

**Comment by nivm** [comment]
NP.
I will update as soon as a pull request is available.
